dnf-behave-tests/features/repoinfo.feature

```diff
@@ -0,0 +1,116 @@
+
+Repo-id            : dnf-ci-thirdparty-updates
+Repo-name          : dnf-ci-thirdparty-updates test repository
+Repo-revision      : 1550000000
+Repo-updated       : Mon Nov 18 20:00:44 2019
+Repo-pkgs          : 6
+Repo-available-pkgs: 6
+Repo-size          : 36 k
+Repo-baseurl       : file:///opt/behave/fixtures/repos/dnf-ci-thirdparty-updates
+Repo-expire        : 172800 second(s) (last: Mon Nov 18 20:00:44 2019)
+Repo-filename      : /tmp/dnf_ci_installroot_ww3liqzs/etc/yum.repos.d/dnf-ci-thirdparty-updates.repo
+Total packages: 295
+"""
+
+Scenario: Repoinfo without arguments and option --all
+   When I execute dnf with args "repoinfo --all"
+   Then the exit code is 0
+    And stdout is
+"""
+Repo-id            : dnf-ci-fedora
+Repo-name          : dnf-ci-fedora test repository
+Repo-status        : enabled
+Repo-revision      : 1550000000
+Repo-updated       : Mon Nov 18 20:00:44 2019
+Repo-pkgs          : 289
+Repo-available-pkgs: 289
+Repo-size          : 2.0 M
+Repo-baseurl       : file:///opt/behave/fixtures/repos/dnf-ci-fedora
+Repo-expire        : 172800 second(s) (last: Mon Nov 18 20:00:44 2019)
+Repo-filename      : /tmp/dnf_ci_installroot_ww3liqzs/etc/yum.repos.d/dnf-ci-fedora.repo
+
+Repo-id            : dnf-ci-fedora-updates
+Repo-name          : dnf-ci-fedora-updates test repository
+Repo-status        : disabled
+Repo-baseurl       : file:///opt/behave/fixtures/repos/dnf-ci-fedora-updates
+Repo-expire        : 172800 second(s) (last: unknown)
+Repo-filename      : /tmp/dnf_ci_installroot_ww3liqzs/etc/yum.repos.d/dnf-ci-fedora-updates.repo
+
+Repo-id            : dnf-ci-thirdparty
+Repo-name          : dnf-ci-thirdparty test repository
+Repo-status        : disabled
+Repo-baseurl       : file:///opt/behave/fixtures/repos/dnf-ci-thirdparty
+Repo-expire        : 172800 second(s) (last: unknown)
+Repo-filename      : /tmp/dnf_ci_installroot_ww3liqzs/etc/yum.repos.d/dnf-ci-thirdparty.repo
+
+Repo-id            : dnf-ci-thirdparty-updates
+Repo-name          : dnf-ci-thirdparty-updates test repository
+Repo-status        : enabled
+Repo-revision      : 1550000000
+Repo-updated       : Mon Nov 18 20:00:44 2019
+Repo-pkgs          : 6
+Repo-available-pkgs: 6
+Repo-size          : 36 k
+Repo-baseurl       : file:///opt/behave/fixtures/repos/dnf-ci-thirdparty-updates
+Repo-expire        : 172800 second(s) (last: Mon Nov 18 20:00:44 2019)
+Repo-filename      : /tmp/dnf_ci_installroot_ww3liqzs/etc/yum.repos.d/dnf-ci-thirdparty-updates.repo
+Total packages: 295
+"""
+
+Scenario: Repoinfo without arguments but with excludes
+   When I execute dnf with args "repoinfo -x=*"
+   Then the exit code is 0
+    And stdout is
+"""
+Repo-id            : dnf-ci-fedora
+Repo-name          : dnf-ci-fedora test repository
+Repo-revision      : 1550000000
+Repo-updated       : Mon Nov 18 20:00:44 2019
+Repo-pkgs          : 289
+Repo-available-pkgs: 0
+Repo-size          : 2.0 M
```

Comment:
The repo size is calculated from available packages => I'm getting 0 here instead of 2.0 M